db/migrate/20220105083927_add_duplicate_match_resolved_to_fraud_match.rb

```diff
@@ -0,0 +1,5 @@
+class AddDuplicateMatchResolvedToFraudMatch < ActiveRecord::Migration[6.1]
+  def change
+    add_column :fraud_matches, :duplicate_match_resolved, :boolean, null: false, default: false
```

Comment:
Maybe this could just be `resolved`? We know it's a match as that's what the model is called.

Reply:
That's very true. Have fixed, thanks.